lib/Rozier/app/components/DrawerItem.vue

```diff
@@ -1,54 +1,33 @@
-            @click.prevent="onAddItemButtonClick"
-            :class="{ 'has-thumbnail': !!thumbnailUrl, 'not-published': published === false }"
-        >
-            <div class="uk-sortable-handle"></div>
-            <div class="border" :style="{ backgroundColor: color }"></div>
-            <figure
-                class="thumbnail"
-                v-if="thumbnailUrl && !isThumbnailProcessable"
-                :style="{ 'background-image': 'url(' + thumbnailUrl + ')' }"
-            ></figure>
-            <figure class="thumbnail" v-else-if="thumbnailUrl && isThumbnailProcessable">
-                <picture>
-                    <source v-if="!thumbnailUrl.endsWith('.webp')" :srcset="thumbnailUrl + '.webp'" type="image/webp" />
-                    <img :src="thumbnailUrl" :alt="name" />
-                </picture>
-            </figure>
-            <div class="names">
-                <p class="parent-name" v-if="parentName">
-                    <template v-if="subParentName">
-                        <span class="sub">
-                            {{ subParentName }}
-                        </span>
-                    </template>
-                    <span class="direct">{{ parentName }}</span>
-                </p>
-                <span class="name">{{ name }}</span>
-                <input type="hidden" :name="inputName" :value="item.id" v-if="inputName && item && item.id" />
-                <div class="links" :class="editItemUrl ? '' : 'no-edit'">
-                    <ajax-link :href="editItemUrl" class="uk-button link uk-button-mini" v-if="editItemUrl">
-                        <i class="uk-icon-rz-pencil"></i> </ajax-link
-                    ><button
-                        type="button"
-                        class="uk-button uk-button-mini link uk-button-danger"
-                        @click.prevent="onRemoveItemButtonClick()"
-                    >
-                        <i class="uk-icon-rz-trash-o"></i>
-                    </button>
-                </div>
-                <button type="button" class="uk-button uk-button-mini link-button">
-                    <div class="link-button-inner">
-                        <i class="uk-icon-rz-plus"></i>
-                    </div>
-                </button>
-            </div>
-        </li>
-    </transition>
+    <div class="rz-drawer__item" @click.prevent="onAddItemButtonClick">
+        <input type="hidden" :name="inputName" :value="item.id" v-if="inputName && item && item.id" />
+        <div class="rz-drawer__item__overtitle" v-if="parentName || subParentName">
+            {{ subParentName ? subParentName : parentName }}
+        </div>
+        <div v-if="name" class="rz-drawer__item__title">{{ name }}</div>
+        <div v-if="thumbnailUrl" class="rz-drawer__item__asset">
+            <picture class="rz-drawer__item__img">
+                <source v-if="!thumbnailUrl.endsWith('.webp')" :srcset="thumbnailUrl + '.webp'" type="image/webp" />
```

Comment:
You do not test `isThumbnailProcessable` anymore. This will lead to error if thumbnail is a SVG